responses/configuration/set.go

```diff
@@ -0,0 +1,67 @@
+package configuration
+
+import (
+	"encoding/json"
+	"reflect"
+	"strings"
+)
+
+/*
+ * SetConfigResponse is for SMA to use when responding to an incoming request to PUT (i.e. UPDATE) a resource.
+ *
+ */
+
+type SetConfigResponse struct {
+	Code           string   `json:"code" yaml:"code,omitempty"`
+	Description    string   `json:"description" yaml:"description,omitempty"`
+	ExpectedValues []string `json:"expectedValues" yaml:"expectedValues,omitempty"`
```

Comment:
For each property can you add a comment to the right briefly describing each one for GoDoc purposes?